### PR TITLE
feat(PPDSC-2330): re-orgnise codemod

### PR DESCRIPTION
**Pull request by Xin00163**
PPDSC-2330

Codemod should be inside a v6.0.0 version folder. This pattern will make the application more tidy in the long run and less confusing.

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2330-reorgnise-codemod
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2330-reorgnise-codemod/storybook

File: lib/codemod/.eslintignore [hunk 1]
```diff
@@ -2,6 +2,6 @@ node_modules
 build
 lcov-report
 
-src/transforms/__tests__/**/actual*.js
-src/transforms/__tests__/**/expected*.js
+src/v6.0.0/__tests__/**/actual*.js
+src/v6.0.0/__tests__/**/expected*.js
```
Comment (mutebg) [inline lib/codemod/.eslintignore:6]: can we just do 
```suggestion
src/v*/__tests__/**/actual*.js
src/v*/__tests__/**/expected*.js
```

**Review by mutebg — APPROVED** [review]
Looks good.

**Review by Xin00163 — COMMENTED** [review]
Published an alpha release and tested with render & virgin. Work as expected

**Review by evgenitsn — APPROVED** [review]

**Review by baburay23 — APPROVED** [review]

**Pull request merged**